src/utilities/codemirror/index.ts

```diff
@@ -25,18 +25,25 @@ export const SeqLanguage = LRLanguage.define({
         Command: customFoldInside,
       }),
       styleTags({
+        Activate: t.namespace,
         Boolean: t.bool,
+        Engine: t.namespace,
+        Epoch: t.namespace,
         GenericDirective: t.namespace,
         Global: t.namespace,
+        GroundBlock: t.namespace,
+        GroundEvent: t.namespace,
         HardwareCommands: t.namespace,
         IdDeclaration: t.namespace,
         ImmediateCommands: t.namespace,
         LineComment: t.comment,
+        Load: t.namespace,
         LoadAndGoDirective: t.namespace,
         LocalDeclaration: t.namespace,
         MetaEntry: t.namespace,
         Model: t.namespace,
         ParameterDeclaration: t.namespace,
+        Request: t.namespace,
```

Comment:
color namespacing is not working in the editor for @GROUND_EPOCH. It is black text instead of green. Not a blocker if you want to merge it in now


@GROUND_EPOCH("GroundEpochName", "+3:00") @REQUEST_BEGIN("request2.name")
      C CMD_0 1 2 3
      @METADATA "cmd_0_meta_name_0" "cmd_0_meta_value_0"
      @MODEL "a" 1 "00:00:00"
      R00:01:40 CMD_1 "1 2 3"
    @REQUEST_END
    @METADATA "req_0_meta_name" "req_0_meta_value"

Reply:
Good catch, addressed in [6ea0ada](https://github.com/NASA-AMMOS/aerie-ui/pull/1362/commits/6ea0ada632b9a533c2504a09612aa1e5df7421b6)